src/main/java/sonia/scm/jenkins/JenkinsConfigurationDto.java

```diff
@@ -39,6 +44,7 @@
   private String url;
   private String username;
   private boolean csrf;
+  private Set<BuildParameterDto> buildParameters;
```

Comment:
A set is used here, but the referenced class does not have isEqual or hash code methods.